modules/core/02-client/keeper/migrations.go

```diff
@@ -29,8 +31,9 @@ func (m Migrator) Migrate2to3(ctx sdk.Context) error {
 
 // Migrate3to4 migrates from consensus version 3 to 4.
 // This migration enables the localhost client.
-func (m Migrator) Migrate3to4(ctx sdk.Context) error {
-	return v7.MigrateLocalhostClient(ctx, m.keeper)
+func (Migrator) Migrate3to4(ctx sdk.Context) error {
```

Comment:
Hmm, I think you might need to have a migration registered. Will quickly check

Reply:
Thought it would fail here https://github.com/cosmos/cosmos-sdk/blob/979f1e839b15358d71008058bbedc56c9d68d19d/types/module/configurator.go#L150, but realized the 02-client migration is wrapped in another func that also does the connection migration, so it indeed can be removed!